tests/providers/google/cloud/hooks/test_dataflow.py

```diff
@@ -1437,30 +1427,61 @@ def test_check_dataflow_job_state_without_job_type(self, job_state, wait_until_f
         result = dataflow_job._check_dataflow_job_state(job)
         assert result == expected_result
 
-    # fmt: off
-    @parameterized.expand([
-        (DataflowJobType.JOB_TYPE_BATCH, DataflowJobStatus.JOB_STATE_FAILED,
-            "Google Cloud Dataflow job name-2 has failed\\."),
-        (DataflowJobType.JOB_TYPE_STREAMING, DataflowJobStatus.JOB_STATE_FAILED,
-         "Google Cloud Dataflow job name-2 has failed\\."),
-        (DataflowJobType.JOB_TYPE_STREAMING, DataflowJobStatus.JOB_STATE_UNKNOWN,
-         "Google Cloud Dataflow job name-2 was unknown state: JOB_STATE_UNKNOWN"),
-        (DataflowJobType.JOB_TYPE_BATCH, DataflowJobStatus.JOB_STATE_UNKNOWN,
-         "Google Cloud Dataflow job name-2 was unknown state: JOB_STATE_UNKNOWN"),
-        (DataflowJobType.JOB_TYPE_BATCH, DataflowJobStatus.JOB_STATE_CANCELLED,
-            "Google Cloud Dataflow job name-2 was cancelled\\."),
-        (DataflowJobType.JOB_TYPE_STREAMING, DataflowJobStatus.JOB_STATE_CANCELLED,
-         "Google Cloud Dataflow job name-2 was cancelled\\."),
-        (DataflowJobType.JOB_TYPE_BATCH, DataflowJobStatus.JOB_STATE_DRAINED,
-            "Google Cloud Dataflow job name-2 was drained\\."),
-        (DataflowJobType.JOB_TYPE_STREAMING, DataflowJobStatus.JOB_STATE_DRAINED,
-         "Google Cloud Dataflow job name-2 was drained\\."),
-        (DataflowJobType.JOB_TYPE_BATCH, DataflowJobStatus.JOB_STATE_UPDATED,
-            "Google Cloud Dataflow job name-2 was updated\\."),
-        (DataflowJobType.JOB_TYPE_STREAMING, DataflowJobStatus.JOB_STATE_UPDATED,
-         "Google Cloud Dataflow job name-2 was updated\\."),
-    ])
-    # fmt: on
```

Comment:
Actually there is quite a few instructions for disable `black`. 
I've just change only related to migration.

But still exists in some modules, we could get rid of them later:

```console
❯ grep -rl '# fmt: off' ./tests/providers | sort | uniq
./tests/providers/google/cloud/hooks/test_dataflow.py
./tests/providers/google/cloud/hooks/test_functions.py
./tests/providers/google/cloud/hooks/test_gcs.py
./tests/providers/google/cloud/hooks/test_life_sciences.py
./tests/providers/google/cloud/hooks/test_mlengine.py
./tests/providers/google/cloud/operators/test_functions.py
./tests/providers/google/cloud/transfers/test_bigquery_to_mssql.py
./tests/providers/google/cloud/transfers/test_bigquery_to_mysql.py
./tests/providers/google/marketing_platform/hooks/test_analytics.py
./tests/providers/google/marketing_platform/hooks/test_campaign_manager.py
./tests/providers/google/marketing_platform/hooks/test_display_video.py
./tests/providers/microsoft/azure/hooks/test_azure_cosmos.py
```